packages/@apphosting/common/src/index.ts

```diff
@@ -1,5 +1,58 @@
 import { spawn } from "child_process";
 
+// Output bundle specifications to be written to bundle.yaml
+export interface OutputBundle {
+  version: "v1alpha";
```

Comment:
prefer to not use alpha/beta versioning. it's being phased out at google and it's added overhead to our own internal api release processes